pylarky/larky.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+
+def evaluate(script, input_file, output_file):
```

Comment:
IMO passing input data via file is a bit leaky abstraction and we should hide files usage behind the interface of this library passing an input data as a string/bytes. So preparing the input file and reading the output one will be responsibility of this library instread of Satellite:
```
def evaluate(script: str, input_data: str) -> str:
   pass
```